lib/plugins/inspec-plugin-manager-cli/test/functional/inspec-plugin_test.rb

```diff
@@ -233,10 +233,10 @@ def test_search_omit_excluded_inspec_plugins
     result = run_inspec_process("plugin search --include-test-fixture inspec-")
     assert_equal 0, result.exit_status, "Search should exit 0"
     assert_includes result.stdout, "inspec-test-fixture", "Search result should contain the test gem"
-    [
-      "inspec-core",
-      "inspec-multi-server",
-    ].each do |plugin_name|
+    %w{
+      inspec-core
+      inspec-multi-server
+    }.each do |plugin_name|
```

Comment:
Personally I prefer `%w[...]`... does chefstyle care about the brackets used?

Reply:
Lets file these issues against Chefstyle and discuss there. I dont think debating the style choices set by Chefstyle in this repo will get the same response.